conf/locale/locale_en-US.ini

```diff
@@ -930,7 +930,7 @@ config.db_ssl_mode_helper = (for "postgres" only)
 config.db_path = Path
 config.db_path_helper = (for "sqlite3" and "tidb")
 config.service_config = Service Configuration
-config.register_email_confirm = Require E-mail Confirmation
+config.register_email_confirm = Require email confirmation
```

Comment:
Whoops, good catch. How about `Require Email Confirmation` to keep the correct casing but still make the e-mail -> email change?